### ISSUE TITLE
admin breaks on CSRF verification in Django 1.10

### ISSUE THREAD

**Issue by rooterkyberian** [django-guardian/django-guardian#444]
Due Django 1.10 changing CSRF every request one has to use `render` instead `render_to_response`
in `guardian/admin.py`.

Here is example how error showed up for me under `runserver` when I tried to edit user object permissions:
```
[07/Aug/2016 23:07:06] "POST /admin/boards/board/114/permissions/user-manage/4/ HTTP/1.1" 403 2502
CENSORED/eggs/Django-1.10-py3.5.egg/django/template/defaulttags.py:65: UserWarning: A {% csrf_token %} was used in a template, but the context did not provide the value.  This is usually caused by not using RequestContext.
  "A {% csrf_token %} was used in a template, but the context "
[07/Aug/2016 23:07:16] "GET /admin/boards/board/114/permissions/user-manage/4/ HTTP/1.1" 200 3571
Forbidden (CSRF token missing or incorrect.): /admin/boards/board/114/permissions/user-manage/4/
[07/Aug/2016 23:07:22] "POST /admin/boards/board/114/permissions/user-manage/4/ HTTP/1.1" 403 2502
```
Please also note while there is no official indication `render_to_response` depreciation, such note exists in documentation of https://docs.djangoproject.com/en/1.10/topics/http/shortcuts/#render-to-response .

Not sure how much other users rely on django guardian admin panel, but for me it effectively blocks Django 1.10 adaptation.

**Comment by tdruez** [comment]
See https://github.com/django-guardian/django-guardian/issues/442 , fixed in https://github.com/django-guardian/django-guardian/pull/443

**Comment by rooterkyberian** [comment]
Yes, thank you, I should have checked it first. I wonder why wasn't the `render_to_response` just replaced with `render` instead of adding that Django version condition, but whatever works I guess.


Now with https://github.com/django-guardian/django-guardian/releases/tag/v1.4.5 it seems to work perfectly with Django 1.10 !;)

**Comment by tdruez** [comment]
> I wonder why wasn't the render_to_response just replaced with render instead of adding that Django version condition, but whatever works I guess.

We need to keep backward compatibility with previous Django versions.
Feel free to submit a PR with a better all-django-version compatible fix ;)

**Comment by QasimK** [comment]
I can confirm that the CSRF issue was fixed when I updated from 1.4.4 -> 1.4.5 👍
